solr/core/src/java/org/apache/solr/metrics/prometheus/core/SolrCoreCacheMetric.java

```diff
@@ -38,7 +37,7 @@ public SolrCoreCacheMetric(
   public SolrCoreMetric parseLabels() {
     String[] parsedMetric = metricName.split("\\.");
     if (dropwizardMetric instanceof Gauge) {
-      labels.put("cacheType", parsedMetric[2]);
+      labels.put("cacheType", parsedMetric[3]);
```

Comment:
Previously index 2 lined up with the sample above on line 33.  But now; I dunno.  I suppose your data example string is now out of date.
Same comment as other source files you edited similarly.

Reply:
Much improved; thanks